cmd/icingadb-migrate/convert.go

```diff
@@ -299,7 +299,7 @@ func convertDowntimeRows(
 			Author:             row.AuthorName,
 			Comment:            row.CommentData,
 			IsFlexible:         types.Bool{Bool: row.IsFixed == 0, Valid: true},
-			FlexibleDuration:   uint64(row.Duration) * 1000,
+			FlexibleDuration:   uint64(max(row.Duration, 0)) * 1000, // #nosec G115 -- IDO duration is a BIGINT; shouldn't be negative
```

Comment:
Is that linter warning really due to the cast now or is it actually the `* 1000` which can also overflow?

In general, I'd prefer some extra checks with a "make linter happy" comment over a "disable linter" comment, as that makes more clear what is handled while the other might accidentally silence other problems in the same line.

Reply:
I have extracted it into a separate block, allowing to get rid of the linter hint.